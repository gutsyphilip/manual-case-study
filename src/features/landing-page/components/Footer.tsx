import React from "react";
import { Link } from "react-router-dom";

import {
  ManualLogo,
  FacebookIcon,
  TwitterIcon,
  LinkedInIcon,
} from "../../../assets/svgs";

import styles from "./landing-page.module.scss";

interface Props {}

const Footer = (props: Props) => {
  return (
    <footer className={styles.footer}>
      <section className={styles.footer_main}>
        <ManualLogo className={styles.footer_main_logo} />
        <nav className={styles.footer_main_nav}>
          <div className={styles.footer_main_nav_group}>
            <header className={styles.footer_main_nav_group_header}>
              <h5 className={styles.footer_main_nav_group_header_title}>
                PRODUCT
              </h5>
            </header>
            <ul className={styles.footer_main_nav_group_list}>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Popular</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Trending</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Guided</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Products</Link>
              </li>
            </ul>
          </div>
          <div className={styles.footer_main_nav_group}>
            <header className={styles.footer_main_nav_group_header}>
              <h5 className={styles.footer_main_nav_group_header_title}>
                Company
              </h5>
            </header>
            <ul className={styles.footer_main_nav_group_list}>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Press Releases</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Mission</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Strategy</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">About</Link>
              </li>
            </ul>
          </div>
          <div className={styles.footer_main_nav_group}>
            <header className={styles.footer_main_nav_group_header}>
              <h5 className={styles.footer_main_nav_group_header_title}>
                INFO
              </h5>
            </header>
            <ul className={styles.footer_main_nav_group_list}>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Support</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Customer Service</Link>
              </li>
              <li className={styles.footer_main_nav_group_list_item}>
                <Link to="">Get Started Guide</Link>
              </li>
            </ul>
          </div>
          <div className={styles.footer_main_nav_group}>
            <header className={styles.footer_main_nav_group_header}>
              <h5 className={styles.footer_main_nav_group_header_title}>
                Follow us
              </h5>
            </header>
            <ul className={styles.footer_main_nav_group_icons}>
              <li className={styles.footer_main_nav_group_icons_icon}>
                <Link to="">
                  <FacebookIcon />
                </Link>
              </li>
              <li className={styles.footer_main_nav_group_icons_icon}>
                <Link to="">
                  <TwitterIcon />
                </Link>
              </li>
              <li className={styles.footer_main_nav_group_icons_icon}>
                <Link to="">
                  <LinkedInIcon />
                </Link>
              </li>
            </ul>
          </div>
        </nav>
      </section>
      <div className={styles.footer_sub}>
        <hr />
        <p className={styles.footer_sub_credits}>
          &copy; 2019 Manual. All rights reserved.
        </p>
      </div>
    </footer>
  );
};

export default Footer;
