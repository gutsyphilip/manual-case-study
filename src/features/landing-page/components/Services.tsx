import React from "react";

import styles from "./landing-page.module.scss";
interface Props {
  servicesList: {
    tagLine: string;
    title: string;
    description: string;
    src: string;
    srcSet: string[];
  }[];
}

const Services: React.FC<Props> = ({ servicesList }) => {
  return (
    <section className={styles.services}>
      <header className={styles.services_header}>
        <h2 className={styles.services_header_title}>What we can help with</h2>
      </header>
      <div className={styles.services_list}>
        {servicesList.map(
          ({ tagLine, title, description, src, srcSet }, index) => {
            return (
              <div
                className={styles.services_list_item}
                key={`${title.replace(" ", "+")}`}
              >
                <img
                  className={styles.services_list_item_image}
                  src={src}
                  srcSet={srcSet.join(", ")}
                  alt={title}
                />
                <article className={styles.services_list_item_caption}>
                  <h6 className={styles.services_list_item_caption_tagLine}>
                    {tagLine}
                  </h6>
                  <h3 className={styles.services_list_item_caption_title}>
                    {title}
                  </h3>
                  <p className={styles.services_list_item_caption_description}>
                    {description}
                  </p>
                </article>
              </div>
            );
          }
        )}
      </div>
    </section>
  );
};

export default Services;
