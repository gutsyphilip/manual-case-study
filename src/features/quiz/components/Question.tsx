import React, { useEffect } from "react";
import styles from "./quiz.module.scss";
import cx from "classnames";

interface Props {
  data: {
    question: string;
    type: string;
    options: { [key: string]: string | boolean }[];
  };
  number: number;
  activeQuestion: number;
  // TODO: Resolve ref type issue
  activeRef?: any;
  userResponses: { [key: string]: string | boolean };
  handleSetResponse: (
    question: string,
    response: string | boolean,
    questionNumber: number
  ) => void;
}

const Question = ({
  data: { options, question, type },
  number,
  activeQuestion,
  handleSetResponse,
  userResponses,
  activeRef,
}: Props) => {
  console.log(activeRef);
  const renderOptions = (): React.ReactNode => {
    return options.map((option, index) => {
      const label: string = Object.keys(option)[0];
      return (
        <div
          className={cx(styles.quizModal_question_options_item, {
            [styles.quizModal_question_options_item__selected]:
              userResponses[question] === options[index][label],
          })}
          key={`${label}`}
          onClick={() => {
            handleSetResponse(question, options[index][label], number);
          }}
        >
          <div
            className={styles.quizModal_question_options_item_label}
            dangerouslySetInnerHTML={{ __html: label }}
          ></div>
          <div className={styles.quizModal_question_options_item_text}>
            {options[index][label]}
          </div>
        </div>
      );
    });
  };

  useEffect(() => {
    if (activeQuestion > 1) {
      // console.log(activeRef.current.offsetTop);
      // activeRef.current.style.top = "100px";
      activeRef?.current?.scrollIntoView({
        behavior: "smooth",
        block: "end",
        inline: "nearest",
      });
    }
  }, [activeRef, activeQuestion]);

  return (
    <div
      className={cx(styles.quizModal_question, {})}
      ref={activeQuestion === number ? activeRef : null}
    >
      <h4
        className={styles.quizModal_question_label}
      >{`Question ${number}`}</h4>
      <h1 className={styles.quizModal_question_title}>{question}</h1>
      <div className={styles.quizModal_question_options}>{renderOptions()}</div>
    </div>
  );
};

export default Question;
