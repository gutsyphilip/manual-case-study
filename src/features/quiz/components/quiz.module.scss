@import "../../../styles/breakpoint";
@import "../../../styles/converter";
@import "../../../styles/variables";

.quizModal {
  overflow-y: scroll;

  &_header {
    position: sticky;
    width: 100%;
    top: 0;
    left: to-rem(0);
    display: flex;
    justify-content: space-between;
    align-items: center;
    background: $white;
    z-index: 3;
    padding: to-rem(20);
    @include md() {
      padding: to-rem(61);
    }
    @include lg() {
      padding: to-rem(70 100 20);
    }

    &_logo {
      width: to-rem(60);
      height: to-rem(55);
    }
    &_closeIcon {
      width: to-rem(60);
      height: to-rem(55);
      cursor: pointer;
    }
  }

  &_question {
    height: 110vh;
    width: 100%;
    transition: all 0.3s cubic-bezier(0.455, 0.03, 0.515, 0.955);
    padding: to-rem( 40 20);

    @include md() {
      padding: to-rem(61);
    }
    @include lg() {
      padding: to-rem(0 100 0);
    }

    &::before {
      content: "";
      height: to-rem(150);
      // width: 100%;
      display: block;

      @include lg() {
        height: to-rem(300);
      }
    }

    &_label {
      text-transform: uppercase;
      color: $manual-light-green;
    }

    &_title {
      font-size: to-rem(26);
      font-family: Montserrat;
      margin-top: to-rem(20);
      @include md() {
        font-size: to-rem(42);
      }
    }

    &_options {
      display: grid;
      grid-template-columns: 1fr 1fr;
      gap: to-rem(10);
      margin: to-rem(50 0);

      @include sm() {
        margin-top: to-rem(80);
        grid-template-columns: repeat(auto-fit, minmax(to-rem(154), 1fr));
        gap: to-rem(50);
      }
      &_item {
        position: relative;
        cursor: pointer;
        display: flex;
        flex-direction: column;
        justify-content: center;
        align-items: center;
        color: rgb(79, 176, 174);
        box-shadow: rgba(79, 176, 174, 0.6) 0px 0px 0px 1px inset;
        cursor: pointer;
        opacity: 1;
        will-change: opacity;
        min-height: to-rem(50);
        height: 100%;
        border-radius: 4px;
        padding: to-rem(3);
        min-width: to-rem(100);
        &:hover {
          background-color: rgba(79, 176, 174, 0.1);
          box-shadow: rgba(79, 176, 174, 0.8) 0px 0px 0px 2px inset;
        }
        &__selected {
          background-color: rgba(79, 176, 174, 0.1);
          box-shadow: rgba(79, 176, 174, 0.8) 0px 0px 0px 2px inset;
          &::before {
            content: "";
            display: block;
            width: 10px;
            height: 10px;
            position: absolute;
            right: 0px;
            top: 0px;
            border-width: 24px;
            border-style: solid;
            border-image: initial;
            border-color: rgb(79, 176, 174) rgb(79, 176, 174) transparent
              transparent;
          }
        }
        &_label {
          img {
            width: to-rem(154);
            height: to-rem(154);
          }
        }
        &_text {
          font-size: to-rem(12);
          @include sm() {
            font-size: to-rem(12);
          }
        }
      }
    }
  }

  &_result {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    text-align: center;
    padding: to-rem(130 20);

    @include md() {
      padding: to-rem(100 61);
    }
    @include lg() {
      padding: to-rem(100 100 0);
    }

    &_icon {
      height: to-rem(80);
      width: to-rem(80);
      margin: to-rem(0) auto to-rem(50);
      text-align: center;
    }

    &_message {
      font-size: to-rem(26);
      font-weight: 500;
      @include md() {
        font-size: to-rem(42);
      }
      a {
        color: $manual-light-green;
      }
    }
  }
}
