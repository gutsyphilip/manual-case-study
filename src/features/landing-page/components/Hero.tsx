import React from "react";
import { Button } from "../../../components/button";
import { ManualLogo } from "../../../assets/svgs";

import styles from "./landing-page.module.scss";

interface Props {
  setShowQuizModal: (value: boolean) => void;
}

const Hero = ({ setShowQuizModal }: Props) => {
  return (
    <header className={styles.hero}>
      <ManualLogo className={styles.hero_logo} />
      <div className={styles.hero_caption}>
        <h1 className={styles.hero_caption_title}>
          BE GOOD <br /> TO YOURSELF
        </h1>
        <p className={styles.hero_caption_description}>
          Weâ€™re working around the clock to bring you a holistic approach to
          your wellness. From top to bottom, inside and out.
        </p>
        <Button
          className={styles.hero_caption_button}
          rounded
          shadow
          variant="themedAlt"
          onClick={() => {
            setShowQuizModal(true);
          }}
        >
          TAKE THE QUIZ
        </Button>
      </div>
    </header>
  );
};

export default Hero;
