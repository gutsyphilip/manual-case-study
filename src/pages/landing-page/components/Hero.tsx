import React from "react";
import { Button } from "../../../components/button";
import { ManualLogo } from "../../../assets/svgs";

import styles from "./landing-page.module.scss";

interface Props {}

const Hero = (props: Props) => {
  return (
    <header className={styles.hero}>
      <ManualLogo className={styles.hero_logo} />
      <div className={styles.hero_caption}>
        <h1 className={styles.hero_caption_title}>BE GOOD TO YOURSELF</h1>
        <p className={styles.hero_caption_description}>
          Weâ€™re working around the clock to bring you a holistic approach to
          your wellness. From top to bottom, inside and out.
        </p>
        <Button className={styles.hero_caption_button} variant="themed">
          TAKE THE QUIZ
        </Button>
      </div>
    </header>
  );
};

export default Hero;
