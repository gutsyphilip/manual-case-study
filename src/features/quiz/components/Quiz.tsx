import React, { Component } from "react";
import { Modal } from "../../../components/modal";
import Question from "./Question";
import {
  ManualLogo,
  FrownIcon,
  SmileIcon,
  CloseIcon,
} from "../../../assets/svgs";

import { questions } from "../../../data/quiz.json";

import styles from "./quiz.module.scss";

interface Props {
  setShowQuizModal: (value: boolean) => void;
}
interface State {
  activeQuestion: number;
  userResponses: { [key: string]: string | boolean };
  isCompleted: boolean;
}

class Quiz extends Component<Props, State> {
  state = {
    activeQuestion: 1,
    userResponses: {},
    isCompleted: false,
  };

  handleSetResponse = (
    question: string,
    response: string | boolean,
    questionNumber: number
  ) => {
    const { userResponses } = this.state;
    this.setState({
      userResponses: { ...userResponses, [question]: response },
    });
    if (questionNumber < 3) {
      this.setState({ activeQuestion: questionNumber + 1 });
    } else {
      this.handleSubmitQuiz();
    }
  };

  handleSubmitQuiz = () => {
    this.setState({ isCompleted: true });
  };

  render() {
    const { activeQuestion, isCompleted, userResponses } = this.state;
    const { setShowQuizModal } = this.props;
    const activeRef = React.createRef();
    return (
      <Modal
        className={styles.quizModal}
        size="cover"
        show={true}
        {...this.props}
      >
        <header className={styles.quizModal_header}>
          <ManualLogo className={styles.quizModal_header_logo} />
          <CloseIcon
            className={styles.quizModal_header_closeIcon}
            onClick={() => {
              setShowQuizModal(false);
            }}
          />
        </header>
        {!isCompleted &&
          questions.map((item, index) => {
            return (
              <Question
                userResponses={userResponses}
                activeRef={activeRef}
                handleSetResponse={this.handleSetResponse}
                key={item.question.replace(" ", "+")}
                data={item}
                number={index + 1}
                activeQuestion={activeQuestion}
              />
            );
          })}

        {isCompleted && (
          <section className={styles.quizModal_result}>
            {Object.values(userResponses).includes(true) ? (
              <>
                <FrownIcon className={styles.quizModal_result_icon} />
                <p className={styles.quizModal_result_message}>
                  Unfortunately, we are unable to prescribe this medication for
                  you. This is because finasteride can alter the PSA levels,
                  which maybe used to monitor for cancer. You should discuss
                  this further with your GP or specialist if you would still
                  like this medication.
                </p>
              </>
            ) : (
              <>
                <SmileIcon className={styles.quizModal_result_icon} />
                <p className={styles.quizModal_result_message}>
                  Great news! We have the perfect treatment for your hair loss.
                  Proceed to{" "}
                  <a
                    href="https://www.manual.co/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    www.manual.co
                  </a>
                  , and prepare to say hello to your new hair!
                </p>
              </>
            )}
          </section>
        )}
      </Modal>
    );
  }
}

export default Quiz;
