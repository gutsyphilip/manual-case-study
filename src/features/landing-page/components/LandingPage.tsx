import React, { Component } from "react";
import Hero from "./Hero";
import Services from "./Services";
import Footer from "./Footer";
import { Quiz } from "../../quiz";
import { SERVICES_LIST } from "../data";

interface Props {}
interface State {
  showQuizModal: boolean;
}

class LandingPage extends Component<Props, State> {
  state = {
    showQuizModal: false,
  };

  handleSetShowQuizModal = (value: boolean): void => {
    this.setState({ showQuizModal: value });
  };

  render() {
    const { showQuizModal } = this.state;
    return (
      <>
        {showQuizModal && (
          <Quiz setShowQuizModal={this.handleSetShowQuizModal} />
        )}
        <Hero setShowQuizModal={this.handleSetShowQuizModal} />
        <main role="main">
          <Services servicesList={SERVICES_LIST} />
        </main>
        <Footer />
      </>
    );
  }
}

export default LandingPage;
