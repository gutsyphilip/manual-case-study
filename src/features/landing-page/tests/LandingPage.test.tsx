import React from "react";
import { render } from "@testing-library/react";
import { LandingPage } from "../components";
import { BrowserRouter } from "react-router-dom";

const LandingPageProps = {};
test("renders landing page", () => {
  const tree = render(
    <BrowserRouter>
      <LandingPage {...LandingPageProps} />
    </BrowserRouter>
  );
  expect(tree.asFragment()).toMatchSnapshot();
});
