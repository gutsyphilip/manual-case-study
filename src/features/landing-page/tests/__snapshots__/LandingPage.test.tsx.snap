// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing page 1`] = `
<DocumentFragment>
  <header
    class="hero"
  >
    <svg
      class="hero_logo"
    >
      manual-logo.svg
    </svg>
    <div
      class="hero_caption"
    >
      <h1
        class="hero_caption_title"
      >
        BE GOOD 
        <br />
         TO YOURSELF
      </h1>
      <p
        class="hero_caption_description"
      >
        We’re working around the clock to bring you a holistic approach to your wellness. From top to bottom, inside and out.
      </p>
      <button
        class="button hero_caption_button button_rounded button_shadow button_themedAlt"
      >
        TAKE THE QUIZ
      </button>
    </div>
  </header>
  <main
    role="main"
  >
    <section
      class="services"
    >
      <header
        class="services_header"
      >
        <h2
          class="services_header_title"
        >
          What we can help with
        </h2>
      </header>
      <div
        class="services_list"
      >
        <div
          class="services_list_item"
        >
          <img
            alt="Hair loss needn’t be irreversible. We can help!"
            class="services_list_item_image"
            src="hair-photo.png"
            srcset="hair-photo@2x.png, hair-photo@3x.png"
          />
          <article
            class="services_list_item_caption"
          >
            <h6
              class="services_list_item_caption_tagLine"
            >
              HAIR LOSS
            </h6>
            <h3
              class="services_list_item_caption_title"
            >
              Hair loss needn’t be irreversible. We can help!
            </h3>
            <p
              class="services_list_item_caption_description"
            >
              Hair loss needn’t be irreversible. There’s a scientifically proven way that’s most effective in keeping and regrowing your hair. It’s all to do with blocking a pesky hormone called DHT. That’s the bad guy behind hair loss. And you can keep him at bay. The choice is yours.
            </p>
          </article>
        </div>
        <div
          class="services_list_item"
        >
          <img
            alt="Erections can be a tricky thing. But no need to feel down!"
            class="services_list_item_image"
            src="sex-photo.png"
            srcset="sex-photo@2x.png, sex-photo@3x.png"
          />
          <article
            class="services_list_item_caption"
          >
            <h6
              class="services_list_item_caption_tagLine"
            >
              Erectile Dysfunction
            </h6>
            <h3
              class="services_list_item_caption_title"
            >
              Erections can be a tricky thing. But no need to feel down!
            </h3>
            <p
              class="services_list_item_caption_description"
            >
              There are plenty of reasons why you might be having difficulty in the erection department. We can help you figure out possible reasons why. And prescribe a pill if needed.
            </p>
          </article>
        </div>
      </div>
    </section>
  </main>
  <footer
    class="footer"
  >
    <section
      class="footer_main"
    >
      <svg
        class="footer_main_logo"
      >
        manual-logo.svg
      </svg>
      <nav
        class="footer_main_nav"
      >
        <div
          class="footer_main_nav_group"
        >
          <header
            class="footer_main_nav_group_header"
          >
            <h5
              class="footer_main_nav_group_header_title"
            >
              PRODUCT
            </h5>
          </header>
          <ul
            class="footer_main_nav_group_list"
          >
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Popular
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Trending
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Guided
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Products
              </a>
            </li>
          </ul>
        </div>
        <div
          class="footer_main_nav_group"
        >
          <header
            class="footer_main_nav_group_header"
          >
            <h5
              class="footer_main_nav_group_header_title"
            >
              Company
            </h5>
          </header>
          <ul
            class="footer_main_nav_group_list"
          >
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Press Releases
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Mission
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Strategy
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                About
              </a>
            </li>
          </ul>
        </div>
        <div
          class="footer_main_nav_group"
        >
          <header
            class="footer_main_nav_group_header"
          >
            <h5
              class="footer_main_nav_group_header_title"
            >
              INFO
            </h5>
          </header>
          <ul
            class="footer_main_nav_group_list"
          >
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Support
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Customer Service
              </a>
            </li>
            <li
              class="footer_main_nav_group_list_item"
            >
              <a
                href="/"
              >
                Get Started Guide
              </a>
            </li>
          </ul>
        </div>
        <div
          class="footer_main_nav_group"
        >
          <header
            class="footer_main_nav_group_header"
          >
            <h5
              class="footer_main_nav_group_header_title"
            >
              Follow us
            </h5>
          </header>
          <ul
            class="footer_main_nav_group_icons"
          >
            <li
              class="footer_main_nav_group_icons_icon"
            >
              <a
                href="/"
              >
                <svg>
                  facebook.svg
                </svg>
              </a>
            </li>
            <li
              class="footer_main_nav_group_icons_icon"
            >
              <a
                href="/"
              >
                <svg>
                  twitter.svg
                </svg>
              </a>
            </li>
            <li
              class="footer_main_nav_group_icons_icon"
            >
              <a
                href="/"
              >
                <svg>
                  linkedin.svg
                </svg>
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </section>
    <div
      class="footer_sub"
    >
      <hr />
      <p
        class="footer_sub_credits"
      >
        © 2019 Manual. All rights reserved.
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
